polkadot/runtime/westend/src/lib.rs

```diff
@@ -1807,6 +1807,9 @@ mod runtime {
 	#[runtime::pallet_index(66)]
 	pub type Coretime = coretime;
 
+	#[runtime::pallet_index(123)]
+	pub type Pov = frame_benchmarking_pallet_pov;
```

Comment:
What is the idea behind adding this?

Reply:
Having it in the kitchensink should be enough, I just added it here since there was some other issue with kitchensink benchmarks.  
Will try to remove again.